homeassistant/components/microbees/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Config flow for microBees integration."""
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from microBeesPy.microbees import MicroBees, MicroBeesException
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import aiohttp_client, config_entry_oauth2_flow
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
**This is still not solved**

Reply:
this is used in the exception

Reply:
Yes, but like I already said in previous comments, you already have to pass a logger to the oauth2 flow, so you can use that logger instead of having 2 loggers defined